src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  .block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    .stopwatch {
+      width: $stopwatchDiam;
+      height: $stopwatchDiam;
+      border: 1vmin dotted $borderColor;
+      border-radius: 100%;
+      position: relative;
+      &__min-hand {
+        height: 20vmin;
+        width: 3vmin;
+        background-color: $blueColor;
+        position: absolute;
+        left: 50%;
+        transform: translate(-50%, 100%);
```

Comment:
The transform 'translate(-50%, 100%)' might not be positioning the element as intended. Verify that it's placing the minute hand correctly at the center of the stopwatch.